docs/android/gradle.md

```diff
@@ -1,26 +1,26 @@
 
 # Gradle Build Instructions - Android
 
-The following instructions describe how to add Product Science instrumentation to an Android application build with Gradle.
+These instructions guide you through the process of integrating Product Science instrumentation into an Android application's Gradle build process.
 
 !!! info
     If your build environment does not allow network access to our servers `https://prod.productscience.app/api/v1/*`, please add to your allowlist.
     If your network settings prevent adding this endpoint, you will be provided with plugin and config archives detailed in sections below.
 
 ## 1. Credentials
 Product Science shared access credentials (`productscience.properties` file) via Bitwarden sent. 
-Please place it in the root directory of your project.
+Please place this file in the root directory of your project.
 
 !!! info
     If your build environment does not allow network access to our servers as specified above, 
     you will be provided with a 'productscience.config.N.zip' archive instead of a .properties file.  
     Copy the entire .zip archive to your workspace directory (do not unzip the archive).
 
 
-## 2. Add Product Science maven repository
-
-In `build.gradle` add the PS maven artifactory to the repositories for project and subprojects:  
+## 2. Add Product Science maven repository to root project's `buildscript`
 
+In the root level **build.gradle** file, add the productscience maven url to the `repositories` inside of the `buildscript` block.
+If you do not have a `buildscript` block in the file, you can add one at the toplevel.
```

Comment:
"top level"

Reply:
I think both are correct but let's put top here to not repeat ourselves